[qat] Conv module

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #23085 [quant] Add fused modules in nn._intrinsic
* **#23084 [qat] Conv module**
* #23082 [reland][qat] Quantization aware training in eager mode

Added Conv module for qat

Differential Revision: [D16379417](https://our.internmc.facebook.com/intern/diff/D16379417/)

jerryzh168 committed:
[qat] Conv module

Added Conv module for qat

Differential Revision: [D16379417](https://our.internmc.facebook.com/intern/diff/D16379417/)

jerryzh168 committed:
Update on "[qat] Conv module"


Added Conv module for qat

Differential Revision: [D16379417](https://our.internmc.facebook.com/intern/diff/D16379417/)

soumith reviewed (approved):
land after CI passes

jerryzh168 committed:
Update on "[qat] Conv module"


Added Conv module for qat

Differential Revision: [D16379417](https://our.internmc.facebook.com/intern/diff/D16379417/)

jerryzh168 committed:
Update on "[qat] Conv module"


Added Conv module for qat

Differential Revision: [D16379417](https://our.internmc.facebook.com/intern/diff/D16379417/)
